_sass/components/_nav.scss

```diff
@@ -194,6 +187,12 @@
   }
 }
 
+@include at-media('desktop') {
+  .desktop-maxw-full {
+    max-width: 100% !important; // overwriting USWDS nav max-width for desktop only
+  }
```

Comment:
I think if you add a snippet like this to [`_variables.scss`](https://github.com/18F/identity-site/blob/main/_sass/_variables.scss), you can turn on USWDS responsive prefixes for max-width utility classes, so you could do `desktop:maxw-full` the same way you're doing `desktop:flex-justify`:

```scss
$max-width-settings: (
  output: true,
  responsive: true,
  active: false,
  focus: false,
  hover: false,
  visited: false,
);
```

Reference: https://github.com/uswds/uswds/blob/v2.13.3/src/stylesheets/settings/_settings-utilities.scss#L371